var/spack/repos/builtin/packages/trexio/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+#
+# To install trexio package in the current envionment
+#
+#     spack install trexio
+#
+# You can edit this file again by typing:
+#
+#     spack edit trexio
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+from spack.package import *
+
+
+class Trexio(AutotoolsPackage):
+    """TREXIO: TREX I/O library."""
+
+    homepage = "https://trex-coe.github.io/trexio"
+    git      = "https://github.com/TREX-CoE/trexio.git"
+    url      = "https://github.com/TREX-CoE/trexio/releases/download/v2.2.0/trexio-2.2.0.tar.gz"
+
+    # notify when the package is updated.
+    maintainers = ['q-posev', 'scemama']
+
+    version('master', branch='master')
+    version('2.0.0', sha256='6eeef2da44259718b43991eedae4b20d4f90044e38f3b44a8beea52c38b14cb4')
+    version('2.1.0', sha256='232866c943b98fa8a42d34b55e940f7501634eb5bd426555ba970f5c09775e83')
+    version('2.2.0', sha256='e6340c424fcea18ae0b643a5707e16005c7576ee21a5aac679fbc132d70b36d9')
```

Comment:
Newer versions are listed first in Spack.

```suggestion
    version('2.2.0', sha256='e6340c424fcea18ae0b643a5707e16005c7576ee21a5aac679fbc132d70b36d9')
    version('2.1.0', sha256='232866c943b98fa8a42d34b55e940f7501634eb5bd426555ba970f5c09775e83')
    version('2.0.0', sha256='6eeef2da44259718b43991eedae4b20d4f90044e38f3b44a8beea52c38b14cb4')
```